src/select/select_value.rs

```diff
@@ -0,0 +1,35 @@
+use crate::select::JsonPathError;
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum SelectValueType {
+    Undef,
+    Null,
+    Bool,
+    Long,
+    Double,
+    String,
+    Array,
+    Dict,
```

Comment:
Why `Dict` instead of `Object`? Seems inconsistent.

Reply:
Changed